src/model/user_model.py

```diff
@@ -211,6 +211,87 @@ class RecommendedTrackModel(db.Model):
     engine_priority = db.Column(db.Integer)
 
 
+class BadRecommendationApplicationModel(db.Model):
+    """
+    BadRecommendationApplication Model for storing bad recommended applications for a user
+    """
+    __tablename__ = "bad_recommendation_application"
+
+    user_id = db.Column(db.Integer, db.ForeignKey(
+        "user.user_id", ondelete="CASCADE"), primary_key=True)
+    app_id = db.Column(db.Integer, db.ForeignKey(
+        "application.app_id", ondelete="CASCADE"), primary_key=True)
+    reason_categorie = db.Column(db.Text, primary_key=True)
+    reason = db.Column(db.Text)# ? , nullable=False)
```

Comment:
remove useless comment
```suggestion
    reason = db.Column(db.Text)
```